linux/system/kernel.sls

```diff
@@ -53,6 +53,21 @@ linux_kernel_module_{{ module }}:
 
 {%- endfor %}
 
+{%- for module_name, module_content in system.kernel.get('module', {}).iteritems() %}
```

Comment:
You could add following condition to determine whether module is enabled or ensure file.absent.
```{%- if module_content.get('enabled', true) %}```